packages/test-ui/src/components/Router.tsx

```diff
@@ -84,16 +113,69 @@ export const Router = ({ web3Provider, signer, chainData }: RouterProps): ReactE
     const addLiquidity = await txManager.addLiquidityFor(liquidityWei, assetId, routerAddress, { value });
     console.log("addLiquidity tx: ", addLiquidity);
     await addLiquidity.wait();
-    const liquidity = await getLiquidity(form.getFieldValue("routerAddress"), form.getFieldValue("assetId"));
-    form.setFieldsValue({
-      currentLiquidity: liquidity,
-    });
+    const liquidity = await getLiquidity(form.getFieldValue("assetId"));
     return liquidity;
   };
 
+  // Refreshes the balances table with human readable values for each asset on current chain.
+  const refreshBalances = async (): Promise<void> => {
+    if (!routerAddress || !(routerAddress.length === 42)) {
```

Comment:
you can do `isValidAddress` from ethers here instead

Reply:
kk